test.js

```diff
@@ -379,6 +379,46 @@ runTests({
 	],
 });
 
+runTests({
+	title: 'have a hideYears option',
+	cases: [
+		[1000 * 60, {hideYears: true}, '1m'],
+		[1000 * 60, {hideYears: false}, '1m'],
+		[1000 * 60 * 67, {hideYears: true}, '1h 7m'],
+		[1000 * 60 * 67, {hideYears: false}, '1h 7m'],
+		[1000 * 60 * 67 * 24 * 465, {hideYears: false}, '1y 154d 6h'],
+		[1000 * 60 * 67 * 24 * 465, {hideYears: true}, '519d 6h'],
+	],
+});
+
+runTests({
+	title: 'have a hideYearsAndDays option',
+	cases: [
+		[1000 * 60, {hideYearsAndDays: true}, '1m'],
+		[1000 * 60, {hideYearsAndDays: false}, '1m'],
+		[1000 * 60 * 67, {hideYearsAndDays: false}, '1h 7m'],
+		[1000 * 60 * 67, {hideYearsAndDays: true}, '1h 7m'],
+		[1000 * 60 * 67 * 24 * 465, {hideYearsAndDays: false}, '1y 154d 6h'],
+		[1000 * 60 * 67 * 24 * 465, {hideYearsAndDays: true}, '12462h'],
+	],
+});
+
+runTests({
+	title: 'have a hideSeconds option',
+	cases: [
+		[(1000 * 60) + 6500, {hideSeconds: false}, '1m 6.5s'],
+		[(1000 * 60) + 6500, {hideSeconds: true}, '1m'],
+	],
+});
+
+runTests({
+	title: 'have hideYearsAndDays,hideSeconds and colonNotation options',
+	cases: [
+		[(1000 * 60 * 60 * 15) + (1000 * 60 * 59) + (1000 * 59) + 543, {hideSeconds: true, hideYearsAndDays: true, colonNotation: true}, '15:59'],
+		[(1000 * 60 * 67 * 24 * 465) + (1000 * 60 * 60 * 15) + (1000 * 60 * 59) + (1000 * 59) + 543, {hideSeconds: true, hideYearsAndDays: true, colonNotation: true}, '12477:59'],
+	],
+});
+
```

Comment:
I would like to see some more tests on how to works together with the other options.

Reply:
Hi, I added numerous test cases. Do you have any specific in mind?